docs/sources/flow/reference/components/prometheus.exporter.unix.md

```diff
@@ -41,6 +41,7 @@ Name | Type | Description | Default | Required
 `procfs_path`              | `string`       | The procfs mountpoint. | `/proc` | no
 `sysfs_path`               | `string`       | The sysfs mountpoint.  | `/sys`   | no
 `rootfs_path`              | `string`       | Specify a prefix for accessing the host filesystem. | `/` | no
+`udev_data_path`           | `string`       | The udev data path.  | `/run/udev/data` | no
```

Comment:
Here says that there's a default value for this, but in the code there's not explicit reference to a default value.

Reply:
I get what you mean. This is set as [default](https://github.com/prometheus/node_exporter/blob/master/collector/paths.go#L29) in node_exporter, so it's inherited. Do you think that we need to add it as a default value in the grafana as well (in the same fashion as [this](https://github.com/grafana/agent/blob/main/pkg/integrations/node_exporter/config.go#L27))?

Reply:
That would be the safer way. Is hard to know how all the integrations behave and if those defaults might change (unlikely but not impossible).